.github/workflows/test.yml

```diff
@@ -283,6 +292,39 @@ jobs:
       run: npm run test
       working-directory: ./integration/substrate
 
+  substrate-subxt:
+    name: Substrate Integration test with subxt
+    runs-on: ubuntu-20.04
+    needs: linux-x86-64
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+      # We can't run substrate as a github actions service, since it requires
+      # command line arguments. See https://github.com/actions/runner/pull/1152
+    - name: Start substrate
+      run: docker run -d -p 9944:9944 paritytech/contracts-ci-linux:production substrate-contracts-node --dev --ws-external
+    - uses: actions/download-artifact@master
+      with:
+        name: solang-linux-x86-64
+        path: bin
+    - run: |
+        chmod 755 ./bin/solang
+        echo "$(pwd)/bin" >> $GITHUB_PATH
+    - uses: actions/download-artifact@master
+      with:
+        name: test-binary
+        path: bin
+    - run: |
+        chmod 755 ./bin/test-binary
+        echo "$(pwd)/bin" >> $GITHUB_PATH        
+    - run: solang compile --target substrate ../substrate/*.sol -o ./contracts/
+      working-directory: ./integration/subxt-tests
+    - run: solang compile --target substrate ../substrate/test/*.sol -o ./contracts/
+      working-directory: ./integration/subxt-tests
+    - name: Deploy and test contracts
+      run: test-binary --test-threads=1
```

Comment:
`cargo test --test-thread=1` ?

Reply:
It's `cargo test -- --test-threads=1` Which is the argument for the test binary to only run one task at a time, this avoids subxt to spam the running substrate node with transaction having the same nonce.